scripts/README.md

```diff
@@ -0,0 +1,43 @@
+# Setup scripts
+
+## Rebuild all contracts
+
+```bash
+./build_all.sh
+```
+
+## Deploy core contracts using master account
+
+### Set master account
+
+```bash
+export MASTER_ACCOUNT_ID=near
+```
+
+### Set network environment
+
+```bash
+export NODE_ENV=testnet
```

Comment:
is this actually a legitimate environment? I think the environment for testnet is `development`

Reply:
Yes, it deploys to `testnet`